symbolic-symcache/src/new/writer.rs

```diff
@@ -26,7 +26,7 @@ pub struct SymCacheConverter {
     string_bytes: Vec<u8>,
     /// A map from [`String`]s that have been added to this `Converter` to [`StringRef`]s, i.e.,
     /// indices into the `string_bytes` vector.
-    strings: IndexMap<String, raw::String>,
+    strings: BTreeMap<String, u32>,
```

Comment:
please update the doc comment. also, I think maybe a `HashMap` would be better here. there is no reason for having the strings be sorted.